spiel/st/m0t1fs_spiel_dix_common_inc.sh

```diff
@@ -101,28 +101,28 @@ spiel_prepare()
 	export SPIEL_OPTS=$SPIEL_OPTS
 	export SPIEL_FIDS_LIST=$SPIEL_FIDS_LIST
 
-	echo SPIEL_OPTS=$SPIEL_OPTS
-	echo SPIEL_FIDS_LIST=$SPIEL_FIDS_LIST
+	echo SPIEL_OPTS="$SPIEL_OPTS"
+	echo SPIEL_FIDS_LIST="$SPIEL_FIDS_LIST"
 
 	# install "motr" Python module required by m0spiel tool
-	cd $M0_SRC_DIR/utils/spiel
+	cd "$M0_SRC_DIR"/utils/spiel
```

Comment:
Do you prefer:
cd "$M0_SRC_DIR/utils/spiel"
?
Please find out which format is better.

Reply:
I have seen earlier if we use the below format, it still shows warning, hence did not touch that
cd "$M0_SRC_DIR/utils/spiel"